Cargo.toml

```diff
@@ -26,3 +26,4 @@ reqwest = { version = "0.10", features= ["blocking"] }
 structopt = "0.3"
 codicon = "3.0"
 colorful = "0.2.1"
+kvm-ioctls = "0.9.0"
```

Comment:
If you're just issuing an ioctl against the `/dev/kvm`, I might err on the side of _not_ pulling in this entire dependency.

Reply:
Have you tried removing this dependency?